source/get-started/pojo-qs.txt

```diff
@@ -0,0 +1,410 @@
+   import com.mongodb.client.result.InsertManyResult;
+   import com.mongodb.client.result.DeleteResult;
+   import com.mongodb.client.result.UpdateResult;
+   import com.mongodb.reactivestreams.client.MongoClient;
+   import com.mongodb.reactivestreams.client.MongoClients;
+   import com.mongodb.reactivestreams.client.MongoCollection;
+   import com.mongodb.reactivestreams.client.MongoDatabase;
+    
+   import org.bson.codecs.configuration.CodecRegistry;
+   import org.bson.codecs.pojo.PojoCodecProvider;
+    
+   import java.util.List;
+    
+   import static com.mongodb.client.model.Filters.*;
+   import static com.mongodb.client.model.Updates.*;
+   import static java.util.Arrays.asList;
+   import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+   import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+- POJO class definitions. The full code is available on GitHub
+  for the ``Person`` and ``Address`` POJOs. The following code
+  demonstrates the main implementation details:
+
+.. code-block:: java
+   
+   public final class Person {
+       private ObjectId id;
+       private String name;
+       private int age;
+       private Address address;
+   
+       public Person() {
+       }
+   
+       public ObjectId getId() {
+           return id;
+       }
+   
+       public void setId(final ObjectId id) {
+           this.id = id;
+       }
+   
+       public String getName() {
+           return name;
+       }
+   
+       public void setName(final String name) {
+           this.name = name;
+       }
+   
+       public int getAge() {
+           return age;
+       }
+   
+       public void setAge(final int age) {
+           this.age = age;
+       }
+   
+       public Address getAddress() {
+           return address;
+       }
+   
+       public void setAddress(final Address address) {
+           this.address = address;
+       }
+       
+       // Rest of implementation
+   }
+   
+   public final class Address {
+       private String street;
+       private String city;
+       private String zip;
+   
+       public Address() {
+       }
+   
+       public String getStreet() {
+           return street;
+       }
+   
+       public void setStreet(final String street) {
+           this.street = street;
+       }
+   
+       public String getCity() {
+           return city;
+       }
+   
+       public void setCity(final String city) {
+           this.city = city;
+       }
+   
+       public String getZip() {
+           return zip;
+       }
+   
+       public void setZip(final String zip) {
+           this.zip = zip;
+       }
+       
+       // Rest of implementation
+   }
+
+Creating a Custom CodecRegistry
+-------------------------------
+
+Before you can use a POJO with the driver, you need to configure the
+``CodecRegistry`` to include a codec to handle the translation to and
+from BSON for your POJOs. The simplest way to do that is to use
```

Comment:
S: I think changing one of the "to"s to "that" makes this sentence a little easier to read

```suggestion
``CodecRegistry`` to include a codec that handles the translation to and
from BSON for your POJOs. The simplest way to do that is to use
```